src/picongpu/include/simulation_defines/unitless/ionizationEnergies.unitless

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2015 Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** \file
+ * This file contains the ionization energies and conversion to corresponding
+ * electric field strengths in atomic units.
+ */
+
+#pragma once
+
+namespace picongpu
+{
+
+    /* \todo We definitely need something nicer to transform an arbitrarily shaped
+     * vector of ionization energies into atomic units
+     */
+    /* ionization energy for ground state hydrogen in atomic units */
+    PMACC_CONST_VECTOR(float_X,1,IONIZATION_ENERGY_AU,
+        float_X(2.18e-18 / SI::ATOMIC_UNIT_ENERGY)
```

Comment:
Please move this magic numbers to `*.param`.
Might be it helps to define the energies relative to a base `BASE_IONIZATION_ENERGY` like [here](https://github.com/ComputationalRadiationPhysics/picongpu/pull/955), than there is no need to normalize each element of `PMACC_CONST_VECTOR(...)`